SignalMessaging/attachments/AttachmentApprovalViewController.swift

```diff
@@ -231,36 +231,53 @@ public class AttachmentApprovalViewController: OWSViewController, CaptioningTool
         // Toolbar flickers in and out if there are errors
         // and remains visible momentarily after share extension is dismissed.
         // It's easiest to just hide it at this point since we're done with it.
-        self.bottomToolbar.isUserInteractionEnabled = false
-        self.bottomToolbar.isHidden = true
+        shouldAllowAttachmentViewResizing = false
```

Comment:
Presenting/dismissing alerts can cause the view controller to become/resign first responder, which has implication for when the keyboard is presented which in turn affects the zoom status of the attachment view.

So we make sure to not fire any more of those animations after the send button has been hit.